CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Check if we are building a pico-sdk based project
+# (or more exactly: if we just got included in a pico-sdk based project)
+if (PICO_SDK_PATH)
+    # If so, load the relevant CMakeLists-file but don't do anything else
+    include(${CMAKE_CURRENT_LIST_DIR}/CMakeLists-PicoSDK.txt)
```

Comment:
This works? I haven't tried using a different filename for _CMakeLists.txt_.

I think this might require pinning CMake to v3.?? (I have to look at the CMake docs again).

Reply:
On second thought, I think we can just rename _CMakeLists-PicoSDK.txt_ to _usePicoSDK.cmake_. Then we can keep the pinned CMake version as is.

Reply:
I only have cmake >= 3.13.5 here, so I can't test the behaviour on CMake 2.x but your proposal sounds reasonable

Reply:
@2bndy5: Renaming done. Same for RF24Network: https://github.com/nRF24/RF24Network/pull/174 and RF24: https://github.com/nRF24/RF24/compare/rp2xxx...kripton:rp2xxx

If you prefer me to squash those 3 commits here into one, I can do so. Or you just squash them ;)

Reply:
thanks for this. I squashed and merged the commits.

As of last night I got the CMake for Linux on RF24 lib to successfully cross-compile for arm; the Linux build CI is also using CMake's CPack module to create installable _.deb_ packages, so compiling the library from source as the primary install procedure may become optional 🚀  There's also a CPack option to create _.rpm_ packages, but it can't be done from a ubuntu OS (I may be mistaken about this).